core/templates/pages/topics-and-skills-dashboard-page/create-new-topic-modal.controller.spec.ts

```diff
@@ -59,21 +59,21 @@ describe('Create new topic modal', function() {
 
 
   it('should check properties set after controller is initialized', function() {
-    var newlyCreatedTopic = NewlyCreatedTopicObjectFactory.createDefault();
+    var newlyCreatedTopic = NewlyCreatedTopic.createDefault();
```

Comment:
[Not related to this PR]
@srijanreddy98, Do you think  `new NewlyCreatedTopic()` should create a topic with default values? (not sure!)

Reply:
By looking at the name of the model, createDefault() method and the constructor method shouldn't be returning different values. Ethier what you said is correct or the naming is off. I feel that we can get rid of this method and have a constructor that returns the default value. This again feels like it should come in a pr that's cleaning up the code. Perhaps the one where we move all these model files into the models' folder.